Week01/Problem04/nkruyer_04.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Jul 12 16:12:57 2018
+
+@author: nkruyer3
+"""
+
+#Week 1 Assignment: Problem 4 - Nick Kruyer
+#correct answer = 906609
+
+#reverses a string
+def rev_str(x):
+    rev = ''
+    x = str(x)
+    for i in range(1,len(x)):
+        rev += x[-i]
+    return rev+x[0]
```

Comment:
clever way to reverse a string! There's actually a really quick way to do this though: `x = x[::-1]`. It works with lists too